packages/ckb-indexer/src/type.ts

```diff
@@ -113,7 +114,7 @@ export class IndexerEmitter extends EventEmitter {
   type?: Script;
   outputData?: HexString | "any";
   argsLen?: number | "any";
-  fromBlock?: bigint;
```

Comment:
fromBlock return in indexer.subscribe as EventEmitter.fromBlock.
cause eventEmitter only used as eventEmitter.on('eventName', handle)

so I think EventEmitter.fromBlock can be private filed. So we can refactor to  ` fromBlock?: JSBI;`.
otherwise we need a  method named `subscibeCompatible`